torch/_inductor/ir.py

```diff
@@ -4015,10 +4028,10 @@ def codegen(self, wrapper):
         wrapper.generate_scatter_fallback(
             x,
             [x, self.constant_args[0], index, src],
-            self.kernel,
+            self.cpp_kernel if V.graph.cpp_wrapper else self.kernel,
```

Comment:
Can we call ```self.codegen_kernel_name``` here?

Reply:
This is because `codegen_kernel_name` doesn't exist in the base class `ExternKernel`, since not all `ExternKernel` has a kernel name. I think @int3 raised a similar question in a different context, `codegen_kwargs`. I will create some other PRs to do some refactoring to the base/child classes here.